### ISSUE TITLE
Update plugin intellij

### ISSUE THREAD

**Issue by dalgarins** [github/codespaces-jetbrains-feedback#132]
- Code space plugin for intellij is outdated, do you have plans to maintain it?

**Comment by omalley** [comment]
Agreed. Gateway is currently completely useless for Github's Codespaces. I guess I'll stick with their web ui or switch to vs code.

**Comment by tuanalumi** [comment]
> I guess I'll stick with their web ui

Do you mean we can connect to GitHub Codespaces without this plugin? If yes, can you please show me how to do that?
